torch/_functorch/_aot_autograd/graph_capture.py

```diff
@@ -244,39 +285,60 @@ def aot_dispatch_base_graph(
         assert maybe_subclass_meta is None, (
             "aot_export_module does not support tensor subclass inputs for now."
         )
-    return fw_module, saved_updated_flat_args_subclasses_desugared, maybe_subclass_meta
+    return (
+        fw_module,
+        saved_updated_flat_args_subclasses_desugared,
+        saved_updated_flat_args_subclasses_desugared_descs,
+        maybe_subclass_meta,
+    )
 
 
 # Has the precondition that there
 # are no duplicate arguments in flat_args (e.g., the same Tensor
 # object never shows up twice.  However, two tensor inputs MAY alias
 # the same storage, so long as they have separate TensorImpls.)
 def aot_dispatch_autograd_graph(
-    flat_fn,
+    flat_fn: TraceFn,
     flat_args: list[Any],
+    flat_args_descs: list[AOTInput],
     aot_config: AOTConfig,
     *,
     fw_metadata: ViewAndMutationMeta,
-) -> tuple[torch.fx.GraphModule, tuple[list[Any], list[Any]], Optional[SubclassMeta]]:
+) -> tuple[
+    torch.fx.GraphModule,
+    tuple[list[Any], list[Any]],
```

Comment:
Should this be list[FxValue] like with aot_dispatch_base_graph?

Reply:
Yes, there are some places I didn't tighten up the types from what they used to be lol